src/scss/vendors/_lineup.scss

```diff
@@ -1,15 +1,14 @@
-// TODO override the following SCSS variables once SCSS import is working again
 $lu_dialog_font_size: 1rem !default;
 $lu_toolbar_font_size: 1rem !default;
 
-// use css because importing sass file from src requires font awesome v4 dependency
-@import '~lineupjs/build/LineUpJS.css';
+$lu_assets: '~lineupjs/src/assets' !default;
+$lu_use_font_awesome: true !default;
```

Comment:
```suggestion
$lu_use_font_awesome: true !default;

// include font awesome files instead of manually adding them
$lu_include_font_awesome: true !default;
```

Reply:
Please notice the variables `$lu_use_font_awesome` and `$lu_include_font_awesome` will be removed with LineUp PR https://github.com/lineupjs/lineupjs/pull/530/files#diff-a8f1570e94da717964e69a68228e193230b7d70fb4c8613f7683eac8b06584b8L87-L92. The PR is targeted for LineUp v5.